lib/runners/processor/haml_lint.rb

```diff
@@ -57,6 +61,12 @@ def default_gem_specs
     def setup
       add_warning_if_deprecated_options([:options])
 
+      if config_linter[:file]
+        add_warning <<~MSG, file: config.path_name
+          The `file` option is deprecated. Use the `target` option instead.
+        MSG
+      end
+
```

Comment:
Isn't sufficient to add an argument to `add_warning_if_deprecated_options`?

```suggestion
      add_warning_if_deprecated_options([:options, :file])
```

Reply:
First, I thought about it. But `add_warning_if_deprecated_options` does not suggest an alternative option, so I thought it's not kindful for users.

Perhaps, we might make `add_warning_if_deprecated_options` customizable, but I think it's out of the scope of this PR. What do you think?

Reply:
Reasonable. Thank you.